.github/workflows/flutter-nightly.yml

```diff
@@ -15,6 +15,14 @@ env:
   # for multiarch gcc compatibility
   VCPKG_COMMIT_ID: "14e7bb4ae24616ec54ff6b2f6ef4e8659434ea44"
   VERSION: "1.2.0"
+  #To make a custom build with your own servers set the below secret values
+  RS_PUB_KEY_VAL: '${{ secrets.RS_PUB_KEY_VAL }}'
+  RENDEZVOUS_SERVER1: '${{ secrets.RENDEZVOUS_SERVER1 }}'
+  RENDEZVOUS_SERVER2: '${{ secrets.RENDEZVOUS_SERVER2 }}'
+  RENDEZVOUS_SERVER3: '${{ secrets.RENDEZVOUS_SERVER3 }}'
+  #ignore signing with key files if values below are set
+  NO_OSX_KEYS: ${{ secrets.NO_OSX_KEYS == 'true' }}
```

Comment:
No need to add `secrets.NO_OSX_KEYS`, you can check available keys `MACOS_P12_BASE64`.

Reply:
The issue with that key is if you do not have a key from apple to use it still tries to use no value and fails, this skips the signing process entirely

Reply:
Updated them to SKIP_APP_KEYS and SKIP_OSX_KEYS so its more clear what it does

Reply:
Ah I understand what you want here, I will change it